content/GStreamerDRPAI.md

```diff
@@ -31,13 +31,23 @@ The plugin also provides you with the following parameters:
 |-----------------------|---------------------|--------:|----------------------------------------------------------------------|
 | **model**             | String              |     --- | The name of the pre-trained model and the directory prefix.          |
 | **multithread**       | Boolean             |    true | Use a separate thread for object detection.                          |
+| **tracking**          | Boolean             |    true | Track detected objects based on their previous locations.            |
```

Comment:
Does this control:
> Track detected objects based on their previous locations.
or rather that a class gets an ID that persists across multiple detections? I'm a bit confused regarding the wording, can you clarify, please?

Reply:
Okay I explain more.